Nestle1904/lowfat/treedown.css

```diff
@@ -182,15 +182,13 @@ w[discontinuous]::after {
 
 /* Apposition */
 
-wg[rule='Np-Appos'] > wg:not(:first-child)::before,
-wg[rule='Np-Appos'] > w:not(:first-child)::before {
+*[apposition='true']::before {
```

Comment:
These CSS changes are simpler, and make no use of GBI rules. As we discussed, the precise language of `@apposition='true'` can be adapted as needed. Unless you have any other immediate preferences, this should probably be clear enough for now.